website/projects/spoke/src/app/core/state/page.state.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+import { Computed, DataAction, StateRepository } from '@ngxs-labs/data/decorators';
+import { NgxsImmutableDataRepository } from '@ngxs-labs/data/repositories';
+import { State } from '@ngxs/store';
+import { Observable } from 'rxjs';
+import { pluck } from 'rxjs/operators';
+
+interface PageStateModel {
+  hasClosedPopup: boolean;
+  allowTelemetry: boolean;
+}
+
+const LOCAL_STORAGE_POPUP_CLOSED_KEY = 'POPUP_CLOSED';
```

Comment:
We don't use POPUP_CLOSED. Remove all references to this. including hasClosedPopup.